operator/cmd/mesh/install.go

```diff
@@ -89,8 +89,8 @@ func InstallCmd(logOpts *log.Options) *cobra.Command {
 		Example: `  # Apply a default Istio installation
   istioctl install
 
-  # Enable grafana dashboard
-  istioctl install --set values.grafana.enabled=true
+  # Enable Envoy's gRPC Access Log Service
+  istioctl install --set meshConfig.enableEnvoyAccessLogService=true
```

Comment:
why the gRPC Access Log Service?  Can we pick something with a little more documentation / support on istio.io?

Reply:
Updated to use tracing flag. Thanks